http/client.go

```diff
@@ -88,9 +159,18 @@ func (c *client) Send(req cmds.Request) (cmds.Response, error) {
 	} else {
 		httpReq.Header.Set(contentTypeHeader, applicationOctetStream)
 	}
-	httpReq.Header.Set(uaHeader, config.ApiVersion)
+	httpReq.Header.Set(uaHeader, c.ua)
+
+	var reqCancel func()
+	if timeoutStr, ok := req.Options[cmds.TimeoutOpt]; ok {
+		timeout, err := time.ParseDuration(timeoutStr.(string))
```

Comment:
We should have a `DurationOption`. Also, see #50. Thoughts?

Reply:
gonna punt on that until later, but I agree